nixos/modules/services/monitoring/prometheus/exporters/pihole.nix

```diff
@@ -6,68 +6,72 @@ let
   cfg = config.services.prometheus.exporters.pihole;
 in
 {
+  imports = [
+    (mkRemovedOptionModule [ "interval"] "This option has been removed.")
+     ({ options.warnings = options.warnings; options.assertions = options.assertions; })
```

Comment:
What's the purpose of doing so, there are a few other examples of this in nixpkgs, all without explanations.

Reply:
Apparently, this is how to do to add `mkRemovedOptionModule`. There are other examples in the same directory.

Reply:
The way below is correct, I believe. And there's no need to do that options.warnings thing.
```
mkRemovedOptionModule [ "prometheus" "exporters" "pihole" "interval" ] "This option has been removed."
```

Reply:
You see that for example: rspamd, doc for the removed option is not correctly rendered on https://search.nixos.org/options?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=services.prometheus.exporters.rspamd

Reply:
Ok, let's try that now.

Reply:
It's failing as you can see...

Reply:
Asked about this on matrix. Yes you were correct, this is something specific to the prometheus modules... Sorry for the noise.

Reply:
This is actually documented in `nixos/modules/services/monitoring/prometheus/exporters.md`, I should have checked that first.

Reply:
No problem :) I just restored the previous state.